scripts/unpack_ddf.py

```diff
@@ -162,6 +162,10 @@ def print_object(printer, msg):
     for descriptor in msg.DESCRIPTOR.fields:
         value = getattr(msg, descriptor.name)
 
+        if not descriptor.label == descriptor.LABEL_REPEATED:
+            if not msg.HasField(descriptor.name):
+                continue
```

Comment:
At runtime, we'll need to check if the structure is actually present in the data, using the "hasX" functionality.